internal/v4/stats_test.go

```diff
@@ -222,7 +222,7 @@ func (s *StatsSuite) TestServerStatsUpdateErrors(c *gc.C) {
 				CharmReference: charm.MustParseReference("~charmers/precise/unknown-23"),
 			}},
 		},
-		expectMessage: "cannot find entity cs:~charmers/precise/unknown-23: entity not found",
+		expectMessage: `cannot find entity for url cs:~charmers/precise/unknown-23: no matching charm or bundle for "cs:~charmers/precise/unknown-23"`,
```

Comment:
Just to clarify: this is needed because of new entities and "entity not found" might not be the case anymore? +1 if so